book/section1/identus-concepts.qmd

```diff
@@ -2,4 +2,14 @@
 title: "Identus Concepts"
 ---
 
-Write Identus Concepts here 
+Identus Application Architecture Diagram
+
+- Trust Registry
```

Comment:
is Trust Registry a concept of Identus? I see this again on SSI Basics which makes more sense